services/learn-card-network/brain-service/serverless.yml

```diff
@@ -21,7 +21,7 @@ constructs:
 
 custom:
     config:
-        CACHE_INSTANCE_SIZE: cache.t2.micro
+        CACHE_INSTANCE_SIZE: ${env:SERVERLESS_CACHE_INSTANCE_SIZE, 'cache.t2.micro'}
```

Comment:
Good catch! This was actually one of the things I wanted to bring up because scouts has a bigger redis instance